generator/lib/common.ml

```diff
@@ -17,14 +17,15 @@ let dbg = Param.mk ~name:"dbg" ~description:["Debug context from the caller"] Ty
 let unit = Param.mk Types.unit
 let task_id = Param.mk ~name:"task_id" Types.string
 
-type uri = string [@@deriving rpcty] [@@doc
-  ["A URI representing the means for accessing the volume data. The interpretation ";
-   "of the URI is specific to the implementation. Xapi will choose which ";
-   "implementation to use based on the URI scheme."]]
+(** A URI representing the means for accessing the volume data. The
+    interpretation of the URI is specific to the implementation. Xapi will
+    choose which implementation to use based on the URI scheme. *)
+type uri = string [@@deriving rpcty]
 
+(** List of blocks for copying. *)
 type blocklist = {
-  blocksize : int
-      [@doc ["size of the individual blocks"]];
-  ranges : (int64 * int64) list
-      [@doc ["list of block ranges, where a range is a (start,length) pair, measured in units of [blocksize]"]];
-} [@@deriving rpcty] [@@doc ["List of blocks for copying"]]
+  blocksize : int ; (** Size of the individual blocks. *)
```

Comment:
`int ;` -> `int;`

Reply:
oh, you have done this everywhere, is that the convention?

Reply:
I followed the notation in https://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#sec359

Reply:
I, too, think a space before a semicolon is unconventional (both in source code and in prose) and was surprised to see it in the manual. But `ocamlformat` also formats code in this way so there seems to be some agreement here.